extraDevfileEntries.yaml

```diff
@@ -0,0 +1,18 @@
+schemaVersion: 1.0.0
+samples:
+  - name: nodejs-basic
```

Comment:
Does each sample need a `version` like we do with stacks?

Reply:
Thinking it over further, I think the answer is no, so gonna resolve this review thread.

Reply:
+1 for no need to have a version for each sample entry. Samples usage are supposed to be one-off clone and detach from the origin so I don't think we need to track the version of that. Same reason why we don't have a version on the starterProject entry.

Reply:
@johnmcollier Can you approve this PR if it's resolved and it looks good to you?